spec/javascripts/fixtures/json/listings-api-listing-preferences.json

```diff
@@ -1 +1 @@
Housing Preference","pdfUrl":null,"order":2,"NRHPDistrict":null,"name":"LP-09777","listingPreferenceID":"a0l0P00001Lx8XZQAZ","listingId":"a0W0P00000F8YG4UAN","description":"For households who are currently paying more than 50% of income for housing costs or are living in public housing or project based Section 8 housing within San Francisco.","appTotal":null},{"unitsAvailable":3,"requiresProof":false,"readMoreUrl":"http://sfmohcd.org/displaced-tenant-housing-preference-program-0","preferenceProofRequirementDescription":null,"preferenceName":"Displaced Tenant Housing Preference (DTHP)","pdfUrl":null,"order":3,"NRHPDistrict":null,"name":"LP-09775","listingPreferenceID":"a0l0P00001Lx8XPQAZ","listingId":"a0W0P00000F8YG4UAN","description":"For households in which at least one member holds a Displaced Tenant Housing Preference Certificate. DTHP Certificate holders are tenants who were evicted through either an Ellis Act Eviction or an Owner Move In Eviction, or have been displaced by a fire. Once all units reserved for this preference are filled, remaining DTHP holders will receive Live/Work preference, regardless of their current residence or work location.","appTotal":null},{"unitsAvailable":4,"requiresProof":false,"readMoreUrl":"http://sfmohcd.org/neighborhood-resident-housing-preference","preferenceProofRequirementDescription":null,"preferenceName":"Neighborhood Resident Housing Preference (NRHP)","pdfUrl":null,"order":4,"NRHPDistrict":null,"name":"LP-09776","listingPreferenceID":"a0l0P00001Lx8XUQAZ","listingId":"a0W0P00000F8YG4UAN","description":"For households in which at least one member either lives within Supervisor district of the project [District #], or within a half-mile of the project.  Requires submission of proof of address.","appTotal":null},{"unitsAvailable":5,"requiresProof":false,"readMoreUrl":"http://sfmohcd.org/housing-preference-programs","preferenceProofRequirementDescription":null,"preferenceName":"Live or Work in San Francisco Preference","pdfUrl":null,"order":5,"NRHPDistrict":null,"name":"LP-09778","listingPreferenceID":"a0l0P00001Lx8XeQAJ","listingId":"a0W0P00000F8YG4UAN","description":"For households in which at least one member lives or works in San Francisco.  Requires submission of proof.  Please note in order to claim Work Preference, the applicant currently work in San Francisco at least 75% of their working hours.","appTotal":null},{"unitsAvailable":null,"requiresProof":false,"readMoreUrl":null,"preferenceProofRequirementDescription":null,"preferenceName":"Works in Public Ed","pdfUrl":null,"order":null,"NRHPDistrict":null,"name":"LP-32885","listingPreferenceID":"a0l0P00001PsqDoQAJ","listingId":"a0W0P00000F8YG4UAN","description":"For households in which at least one member was a resident of the Alice Griffith housing development. This includes baseline and current residents that lived in the targeted redevelopment site on or after the time of application for Choice Neighborhoods of October 26, 2010.","appTotal":null}]}
```

Comment:
Changed Alice Griffith named preference to "Works in public ed" here to get it to work with a test that was expecting a custom preference.